scss/components/_case-list-table.scss

```diff
@@ -0,0 +1,139 @@
+.civicase__table--caselist {
+  .fixed-col {
+    .label {
+      font-size: 0.8em;
+    }
+  }
+
+  .crm_notification-badge {
+    line-height: 18px;
+    padding: 0 10px;
+  }
+
+  &__header {
+    &.affix {
```

Comment:
What is the usage of `affix` class?

Reply:
and all class names needs to be converted to BEM.

Reply:
`affix` class is a part of Bootstrap affix library https://www.w3schools.com/bootstrap/bootstrap_ref_js_affix.asp This can't be converted into BEM.